opencl/source/cl_device/cl_device_caps.cpp

```diff
@@ -198,6 +198,21 @@ void ClDevice::initializeCaps() {
         deviceExtensions += sharingFactory.getExtensions(driverInfo.get());
     }
 
+    PhysicalDevicePciBusInfo pciBusInfo;
+
+    if (driverInfo) {
+        pciBusInfo = driverInfo->getPciBusInfo();
+    }
+
+    deviceInfo.pciBusInfo.pci_domain = pciBusInfo.pciDomain;
+    deviceInfo.pciBusInfo.pci_bus = pciBusInfo.pciBus;
+    deviceInfo.pciBusInfo.pci_device = pciBusInfo.pciDevice;
+    deviceInfo.pciBusInfo.pci_function = pciBusInfo.pciFunction;
+
+    if (isPciBusInfoValid()) {
+        deviceExtensions += "cl_intel_pci_bus_info ";
+    }
```

Comment:
I cannot see ULT that verifies this extension string is actually added

Reply:
The tests have been added. Please see new tests in [opencl/test/unit_test/device/device_caps_tests.cpp](https://github.com/intel/compute-runtime/pull/374/files#diff-705baf1d8b31c58ef2b62f9d06457ae5352300c339ddacc5f00fe6efd4151f3c)